code/drasil-example/Drasil/GamePhysics/Unitals.hs

```diff
@@ -286,12 +287,17 @@ mmntOfInCons   = constrained' QP.momentOfInertia    [nonNegativeConstraint] (dbl
 gravAccelCons  = constrained' QP.gravitationalConst [] (dbl 9.8)
```

Comment:
Newton's gravitational constant is NOT the same thing as the acceleration due to gravity.  The value should be 6.67430(15)×10^{−11} (in the units used).  You can see this at https://en.wikipedia.org/wiki/Gravitational_constant .  @JacquesCarette is correct that this value, and the acceleration due to gravity on the Earth (9.81 m^2/2) should be stored somewhere in Drasil that all of the examples can access.  I don't know how we add  provenance information (I've been lumping all of this kind of information into the term "rationale"), but we should definitely start thinking about this.  It keeps coming up, which is a sign to me that we are on the right track.  :smile: